.github/workflows/fork_sync.yml

```
@@ -0,0 +1,18 @@
+name: Sync Fork
+
+on:
+  schedule:
+    - cron: '*/30 * * * *' # every 30 minutes
```

Let's check every morning for changes
https://crontab.guru/

```suggestion
    - cron: '0 6 * * *' # every day at 6am
```